Some thoughts for auditive impaired (#4)
hackathon-accessibility/hackathon-accessibility.github.io: docs/deaf.md

## Invisible disabilites

Special issues for impaired
- they must have the courage to ask for help and not pretend to be ok
- foreign languages are hard to understand and speak

Special issues for others

> - theycannot know who is deaf 

Reviewer: ```suggestion
- they cannot know who is deaf 
```

- they never know if and when a deaf person understands correctly

### Auditive Impairment

> We distinguish between deaf (people who becae deaf but can read / write / speak) and Deaf (people who are deaf from birth and commiucate with sign language).

Reviewer: ```suggestion
We distinguish between deaf (people who became deaf but can read / write / speak) and Deaf (people who are deaf from birth and communicate with sign language).
```

- they cannot follow discussions in a group

> - they cannot at the same time read subtitles (closed captions) - sing translator and follow for example a video or shared screen

Reviewer: ```suggestion
- they cannot at the same time read subtitles (closed captions) - sign translator and follow for example a video or shared screen
```

- they cannot code and listen to someone at the same time 

#### Beeing deaf means

Presentations:

> As speaker I want not manxy slides (death per powerpoint) but deaf peple like many slides. 

Reviewer: ```suggestion
As speaker I want not many slides (death per powerpoint) but deaf peple like many slides. 
```

You can do this: 
- Prepare a doc for auditive impaired people where they can read during presentation

> - Have a translator who writes simultaneously and the deaf person can read (ther are apps .. but don't know what)

Reviewer: ```suggestion
- Have a translator who writes simultaneously and the deaf person can read (there are apps .. but don't know what)
```

Team work
- Speak slowly
- Avoid standing with your back to the light (lips are harder to see in backlight).

Organisator: 
- reading from lips is possible if there is bright light

#### Beeing Deaf means

Presentation

> - someone who can sign

Reviewer 2: ```suggestion
- someone who can sign in the language spoken.
```
